src/tasks/AppleAppBuilder/Templates/runtime.m

```diff
@@ -307,6 +307,7 @@
     mono_jit_set_aot_mode (MONO_AOT_MODE_INTERP);
 #else
     mono_jit_set_aot_mode (MONO_AOT_MODE_FULL);
+    setenv("MONO_AOT_MODE", "aot", TRUE);
```

Comment:
Can this be done in a separate PR since this is Apple and not Android?

Reply:
It was discussed a bit earlier that it'd make sense to unite an effort in part of disabling the tests which identically fail on android/ios. I introduced PlatformDetection.IsNotDeviceAOT which uses PlatformDetection.IsMonoAOT relying on this variable.